Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_cff.py

```diff
@@ -1,4 +1,9 @@
 import FWCore.ParameterSet.Config as cms
+import HLTrigger.HLTfilters.htlHighLevel_cfi
 
-seqALCARECOEcalTrg = cms.Sequence()
+ecalTrgHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
+#    eventSetupPathsKey='AlCa_EcalPhiSym*', # this is the HLT path that can be used                                                                                                          
+     eventSetupPathsKey='AlCaEcalTrg',
```

Comment:
I'm in doubt about the eventSetupPathsKey. Should that be a new trigger bit or not?
The HLT and L1 bits should be those of the ZeroBias PD. If I have understood correctly how the hltHighLevel filter works, it can used either the HLTPath or the eventSetupPathsKey. The AlCaEcalTrg bit can point to the AlCa_EcalPhiSym* HLT path